STRATEGIC-INITIATIVES.md

```diff
@@ -0,0 +1,46 @@
+# Strategic Initiatives
+
+At any one time the Node.js project has a number of strategic initiatives
+underway.  With the less-tangible approach to the work that the Community 
+Committee does, having a way to track this work is important.
+
+For each strategic initiative, the Community Committee's goal is to have a
+single point of reference - a champion - that understands what's happening
+with the initiative to the fullest extent posssible, and can report on the
+initiative as needed.
+
+A review of the initiatives will be a standing item on the Community Committee agenda (even
+if the update is 'nothing new') as a way to ensure they are active
+and have the support needed.
+
+# Current Initiatives
+
+| Initiative         | Champion                        | Links                                                            |
+|--------------------|---------------------------------|------------------------------------------------------------------|
+| NodeTogether       | Rachel White                    | https://github.com/nodejs/community-committee/issues/63          |
+| User Feedback      | Michael Dawson                  | https://github.com/nodejs/user-feedback                          |
+| Community Events   | PatrickHeneise                  | https://github.com/nodejs/community-events                       |
+| Node.js Collection | Tierney Cyren                   | https://github.com/nodejs/nodejs-collection                      |
```

Comment:
How does this differ from the work of the Evangelism WG? Do we want to highlight all the sub-initiatives or collaborations CommComm teams do as strategic? (Note: I plan on referencing this decision to make a recommendation as to whether @nodejs/user-feedback would request that @nodejs/enterprise-advisory-group be added as strategic.)

Reply:
Ah, thank you! I need to add the Evangelism WG to this document as well, as (in my opinion - happy to have members dispute this) it is definitely an _initiative_ of the CommComm.

Reply:
Just re-read your comment and wanted to follow up - If you're referring to the Node.js Collection, the Node.js Collection is under the CommComm and not a part of the Evangelism WG. While Evangelism WG members are involved and helped kicked it off, it's overseen by the CommComm (ref: [nodejs/nodejs-collection#about-this-team](https://github.com/nodejs/nodejs-collection#about-this-team))